### PR TITLE
adds a delete dog route.

### LINKED ISSUE #25: Delete dog route needed
There's no way to delete a dog!

### PR DESCRIPTION
**Pull request by jonfinger**
This adds a delete dog route. Given a dog_id, delete that dog record and return a success response.

I also added/updated some of the response codes where I forgot to upkeep them.

### CODE REVIEW

File: sniffr/dog_routes/dog_routes.py [hunk 1]
```diff
@@ -134,4 +134,25 @@ def post_dog():
         response = process_record(queried_dog)
         response['breed'] = queried_dog.breed.breed_name
 
-        return response
+        return response, 201
+
+#
```
Comment (allie-rae) [inline sniffr/dog_routes/dog_routes.py:138]:
```suggestion
```

**Review by DeusExMashima — APPROVED** [review]
Looks like a good boy to me

File: sniffr/dog_routes/dog_routes.py [hunk 2]
```diff
@@ -134,4 +134,24 @@ def post_dog():
         response = process_record(queried_dog)
         response['breed'] = queried_dog.breed.breed_name
 
-        return response
+        return response, 201
+
```
Comment (allie-rae) [inline sniffr/dog_routes/dog_routes.py]:
```suggestion
#
```

**Review by allie-rae — APPROVED** [review]
WEEEEEWWW

**Review by squashgray — APPROVED** [review]
all gucci

**Review by danimal-johnson — APPROVED** [review]

**Pull request merged**